src/constants.ts

```diff
@@ -86,3 +86,5 @@ var resources: any = {
 };
 
 _.set(window, 'OPENSHIFT_CONSTANTS.CATALOG_HELP_RESOURCES', resources);
+
+_.set(window, 'OPENSHIFT_CONSTANTS.NEW_PROJECT_MESSAGE', "Contact your system administrator at projectname@example.com to request a project.");
```

Comment:
In that case, we should make the default message something real because the custom message is not required.